garage/tf/q_functions/discrete_mlp_q_function.py

```diff
@@ -0,0 +1,52 @@
+"""Discrete MLP QFunction."""
+import tensorflow as tf
+
+from garage.misc.overrides import overrides
+from garage.tf.core.mlp import mlp
+from garage.tf.q_functions import QFunction
+
+
+class DiscreteMLPQFunction(QFunction):
+    """
+    Discrelte MLP Functoin class.
+
+    This class implements a q value network to predict q based on the input
```

Comment:
Please be consistent with literature in your comments

Q-value network. Predicts the Q-value based on....